model/src/main/java/org/mskcc/cmo/metadb/model/MetaDbSample.java

```diff
@@ -0,0 +1,194 @@
+package org.mskcc.cmo.metadb.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import org.mskcc.cmo.shared.Library;
+import org.mskcc.cmo.shared.QcReport;
+import org.mskcc.cmo.shared.SampleManifest;
+import org.neo4j.ogm.annotation.GeneratedValue;
+import org.neo4j.ogm.annotation.Id;
+import org.neo4j.ogm.annotation.NodeEntity;
+import org.neo4j.ogm.annotation.Relationship;
+import org.neo4j.ogm.annotation.typeconversion.Convert;
+import org.neo4j.ogm.id.UuidStrategy;
+import org.neo4j.ogm.typeconversion.UuidStringConverter;
+
+@NodeEntity
+public class MetaDbSample extends SampleManifest {
+    @Id @GeneratedValue(strategy = UuidStrategy.class)
+    @Convert(UuidStringConverter.class)
+    private UUID uuid;
+    @Relationship(type = "IS_ALIAS", direction = Relationship.INCOMING)
+    private List<SampleAlias> sampleAliases;
+    @Relationship(type = "HAS_SAMPLE", direction = Relationship.INCOMING)
+    private MetaDbPatient patient;
+    @Relationship(type = "HAS_METADATA", direction = Relationship.OUTGOING)
+    private SampleManifestEntity sampleManifestEntity;
```

Comment:
this should be `List<SampleManifestEntity>` since we can expect more than one version of the sample manifest data for a given sample